### PR TITLE
timeout for requests

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: osbs/http.py [hunk 1]
```diff
@@ -43,15 +43,19 @@ def __init__(self, verbose=False):
         self.verbose = verbose
 
     def get(self, url, **kwargs):
+        kwargs.setdefault('timeout', HTTP_REQUEST_TIMEOUT)
```
Comment (vrutkovs) [inline osbs/http.py]: All of those call `self.request`, it would be best to be set there

Comment (rcerven) [reply osbs/http.py]: yeah I've just realized it now :)

File: osbs/constants.py [hunk 2]
```diff
@@ -88,6 +88,9 @@
 # HTTP methods that we should retry on
 HTTP_RETRIES_METHODS_WHITELIST = ['GET', 'PUT', 'POST', 'DELETE']
 
+# requests timeout in seconds
+HTTP_REQUEST_TIMEOUT = 120
```
Comment (twaugh) [inline osbs/constants.py]: Please can we increase this to something like 600? cf. https://github.com/openshift/origin/blob/v3.6.0/pkg/cmd/server/api/types.go#L716-L718

**Comment by rcerven** [comment]
increased timeout, and added catching 'Timeout'

as for the logs()
HttpSession will catch RetryError and raise OsbsNetworkException,
and logs() are catching only OsbsResponseException which would be raised by HttpResponse, if there wouldn't be any exception in HttpSession

**Comment by rcerven** [comment]
test added

**Comment by vrutkovs** [comment]
Release notes updated

**Pull request merged**